share/module_check_a_mundo.F

```diff
@@ -401,6 +401,18 @@ END FUNCTION bep_bem_nbui_max
          END IF
       ENDDO
 
+!-----------------------------------------------------------------------
+! Check that channel irrigation is run with Noah
+!-----------------------------------------------------------------------
+      DO i = 1, model_config_rec % max_dom
+         IF ( model_config_rec % sf_surface_physics(i) .EQ. LSMSCHEME .AND.  &
+             model_config_rec % sf_surf_irr_scheme(i) .EQ. CHANNEL ) THEN
+              wrf_err_message = '--- ERROR: irrigation Opt 1 works only with Noah-LSM'
+              CALL wrf_message ( wrf_err_message )
+         count_fatal_error = count_fatal_error + 1
+         END IF
+      ENDDO
+
```

Comment:
@arjanna @dudhia 
Is this IF test backwards? The comments say we must have LSM with channel. The IF test fails with that combination.

Reply:
In previous calls (and in the Registry.EM_COMMON.var) "lsmscheme" is Noah. If it's wrong like this, which would be the correct? Thanks!

Reply:
@arjanna @dudhia 
This IF test says
```
if ( using NOAH and CHANNEL ) then
   FAIL
```
Is this correct?
I though the logic would be something like this:
```
if ( using NOAH and CHANNEL ) then
   everything is OK
else
   FAIL
```

Reply:
yes sorry, I noticed now the mistake. I fixed it and I am checking the output now. I will commit the change with the new push.

Reply:
Yep, good to go